### PR TITLE
Upgrade to adminlte3

### PR DESCRIPTION
**Pull request by farridav**
Upgrade to AdminLTE3, and upgrade all of its dependencies, basically re-write all templates, and run through entire admin

## TODO
- [x] Run test app in supported Django versions, provide compatibility fixes
- [x] Add supported python and Django versions to README [example [here](https://shields.io/)]
- [x] Cleanup unused static files
- [x] run through main breakpoints (Small, Medium & large screen sizes) and fix where necessary
- [x] Make new screenshots

### CODE REVIEW

File: jazzmin/static/adminlte/plugins/jquery-ui/AUTHORS.txt [hunk 1]
```diff
@@ -0,0 +1,333 @@
...
+Dan Strohl <dan@wjcg.net>
+Maksim Ryzhikov <rv.maksim@gmail.com>
+Amine HADDAD <haddad@allegorie.tv>
+Amanpreet Singh <apsdehal@gmail.com>
+Alexey Balchunas <bleshik@gmail.com>
+Peter Kehl <peter.kehl@gmail.com>
+Peter Dave Hello <hsu@peterdavehello.org>
+Johannes Schäfer <johnschaefer@gmx.de>
+Ville Skyttä <ville.skytta@iki.fi>
+Ryan Oriecuia <ryan.oriecuia@visioncritical.com>
```
Comment (TysonRV) [inline jazzmin/static/adminlte/plugins/jquery-ui/AUTHORS.txt:333]: I guess we can get rid of this 😄

File: jazzmin/utils.py [hunk 2]
```diff
@@ -20,16 +23,34 @@ def order_with_respect_to(first, reference):
...
+            url = reverse("admin:{app_label}_{model_name}_change".format(
+                app_label=app_label, model_name=model_name
+            ), args=(instance.pk,))
+
     except NoReverseMatch:
-        logger.error('Could not reverse {model}, it must be in <app_label>.<model_name> format'.format(model=model))
-        return None
+        logger.error(f'Couldnt reverse url from {instance}')
+
+    return f'{url}?{urlencode(kwargs)}'
```
Comment (TysonRV) [inline jazzmin/utils.py:53]: Mmm.. this f-strings will fail with Python < 3.6

**Comment by TysonRV** [comment]
@farridav This looks great! Will try to tick those boxes off on the weekend and further review it

**Comment by farridav** [comment]
Feel free to commit to this branch, im now fully up to date

**Comment by farridav** [comment]
Get setup, and click through to every possible part of the admin, 

See here the HTML for adminlte components:
https://adminlte.io/themes/v3/index3.html

See the current implementation and template context we have available:
https://github.com/farridav/django-jazzmin/tree/master/jazzmin

Clone this, and copy over dist files/folders when needed:
https://github.com/ColorlibHQ/AdminLTE

**Comment by farridav** [comment]
Im wondering if we should skip to the Cleanup & Finishing tasks to get a release out, and work on other items as improvements...

**Comment by TysonRV** [comment]
> Im wondering if we should skip to the Cleanup & Finishing tasks to get a release out, and work on other items as improvements...

That sounds good to me

**Comment by farridav** [comment]
@TysonRV what are our chances of getting this finished up ? its snowballing ;)

**Pull request merged**